twint/user.py

```diff
@@ -18,17 +18,21 @@ def __init__(self):
 # ur object must be a json from the endpoint https://api.twitter.com/graphql
 def User(ur):
     logme.debug(__name__ + ':User')
-    if 'data' not in ur and 'user' not in ur['data']:
+    if 'data' not in ur or 'user' not in ur['data']:
         msg = 'malformed json! cannot be parsed to get user data'
+        if 'errors' in ur:
+            msg = ur['errors'][0]['message']
         logme.fatal(msg)
-        raise KeyError(msg)
```

Comment:
Shouldn't this still be a KeyError since the key is not in the dictionary? Or maybe it should be the case if `errors` in `ur`, raise Runtime, then if the conditions on line 21 are true, then KeyError?

Reply:
Yes you are correct.

I should have seperated these two errors.

Every once in a while, it's possible that `ur` passed to `User` isn't _json_ at all.
So in that case we need to raise a RuntimeError (ideally twint should have custom Exceptions, but among other things those haven't been implemented yet) 

In the other case though, when the `ur` is infact a _json_ ,  the raised error should be a `KeyError`.